tests/MSBuildDeviceIntegration/Tests/PerformanceTest.cs

```diff
@@ -332,7 +272,7 @@ public void Build_XAML_Change (bool produceReferenceAssembly, bool install)
 			var lib = new DotNetStandard {
 				ProjectName = "MyLibrary",
 				Sdk = "Microsoft.NET.Sdk",
-				TargetFramework = "netstandard2.0",
+				TargetFramework = "net8.0", // Vanilla project
```

Comment:
why this change? netstandard2.0 is still supported? should we test both?

Reply:
`netstandard2.0` doesn't default to `ProduceReferenceAssembly=true`. This is just an attempt to use something more modern, closer to a MAUI project.

This really just tests the timing of an incremental build of a non-Android project, and fast deploying it.